examples/kubernetes/basic/variables.tf

```diff
@@ -1,5 +1,6 @@
 variable "prefix" {
   description = "A prefix used for all resources in this example"
+  default     = "tomdev"
```

Comment:
Maybe `hashidev` would be a better default value?

Reply:
🤦‍♂️ didn't mean to commit this - will revert